gcp/modules/couchdb/main.tf

```diff
@@ -150,6 +152,11 @@ resource "null_resource" "couchdb_destroy_pvcs" {
         for PVC in $(kubectl get pvc --namespace ${var.release_namespace} -o json | jq --raw-output '.items[] | select(.metadata.name | startswith("database-storage-couchdb")) | .metadata.name'); do
           timeout -t 600 kubectl --namespace ${var.release_namespace} delete --ignore-not-found --grace-period=300 pvc $PVC
         done
+        # Some times GCP disks persist even after PVC destruction, removing them with gcloud to not leave stale resources behind
+        # More info: https://issues.gpii.net/browse/GPII-3743
+        for DISK in $(gcloud compute disks list --filter description:couchdb --format json | jq -r -c .[].name); do
+          timeout -t 600 gcloud beta compute disks delete $DISK --quiet --region ${var.infra_region} 2> /dev/null || true
```

Comment:
Why is the `beta` needed here?

Reply:
It allows to delete disks by providing region instead of zone. There is no easy way to extract zone from `disks list` json output.